Docs 2940 scale 22.02 reorg and update replication articles (#1539)
truenas/documentation: content/SCALE/SCALETutorials/DataProtection/Replication/LocalReplicationSCALE.md

The wizard takes new snapshots of the sources when no existing source snapshots are found.  
Enabling **Recursive** replicates all snapshots contained within the selected source dataset snapshots.
Local sources can also use a naming schema to identify any custom snapshots to include in the replication.
A naming schema is a collection of [strftime](https://www.freebsd.org/cgi/man.cgi?query=strftime) time and date strings and any identifiers that a user might have added to the snapshot name.

![TasksReplicationTasksAddLocalSourceSCALE](/images/SCALE/RepWizardLocalSourceSCALE.png "Replication with Local Source")

Set the destination to the local system and define the path to the storage location for replicated snapshots.
When manually defining the destination, be sure to type the full path to the destination location.

![TasksReplicationTasksAddLocalDestSCALE](/images/SCALE/RepWizardLocalDestSCALE.png "Local Destination")

TrueNAS suggests a default name for the task based on the selected source and destination locations, but you can type your own name for the replication.
You can load any saved replication task into the wizard to make creating new replication schedules even easier.

> You can define a specific schedule for this replication or choose to run it immediately after saving the new task.
> Unscheduled tasks are saved in the replication task list and you can run saved tasks manually or edit later to add a schedule.

Reviewer: ```suggestion
You can define a specific schedule for this replication or choose to run it immediately after saving the new task.
TrueNAS saves unscheduled tasks in the replication task list. You can run saved tasks manually or edit them later to add a schedule.
```

The destination lifetime is how long copied snapshots are stored in the destination before they are deleted.
We usually recommend defining a snapshot lifetime to prevent storage issues.
Choosing to keep snapshots indefinitely can require you to manually clean old snapshots from the system if or when the destination fills to capacity.

![TasksReplicationTasksAddLocalSourceLocalDestCustomLife](/images/CORE/12.0/TasksReplicationTasksAddLocalSourceLocalDestCustomLife.png "Custom Lifetime")

Clicking **START REPLICATION** saves the new task and immediately attempts to replicate snapshots to the destination.
When TrueNAS detects that the destination already has unrelated snapshots, it asks to delete the unrelated snapshots and do a full copy of the new snapshots.

> This can delete important data, so be sure any existing snapshots can be deleted or are backed up in another location.

Reviewer: ```suggestion
This can delete important data, so ensure you can delete any existing snapshots or back them up in another location.
```

> The simple replication is added to the replication task list and shows that it is currently running.
> Clicking the task state shows the replication log with an option to download the log to your local system.

Reviewer: ```suggestion
TrueNAS adds the simple replication to the replication task list and shows that it is currently running.
Clicking the task state shows the replication log with an option to download the log to your local system.
```

![TasksReplicationTasksLogSCALE](/images/SCALE/RepLogSCALE.png "Replication Log")

> To confirm that snapshots are replicated, go to **Storage > Snapshots >Snapshots** and verify the destination dataset has new snapshots with correct timestamps.

Reviewer: ```suggestion
To confirm that snapshots are replicated, go to **Storage > Snapshots > Snapshots** and verify the destination dataset has new snapshots with correct timestamps.
```
